app/client/widgets/AttachmentsWidget.ts

```diff
@@ -142,30 +160,83 @@ export class AttachmentsWidget extends NewAbstractWidget {
     });
   }
 
+  private _setUploadingState(rowId: UIRowId, uploading: boolean): void {
+    const timeouts = this._uploadingTimeouts;
+    const states = this._uploadingStatesObs.get();
+    if (timeouts[rowId]) {
+      clearTimeout(timeouts[rowId]);
+    }
+    if (uploading) {
+      timeouts[rowId] = window.setTimeout(() => {
```

Comment:
```suggestion
      timeouts[rowId] = window.setTimeout(() => {
        if (this.isDisposed()) { return; }
```

Being a little paranoid here, but this may prevent a JS error if the component is disposed in the middle of an upload.